gradle/libs.versions.toml

```diff
@@ -22,7 +22,6 @@ nessieClientVersion = "0.77.1" # Must be in sync with Nessie version in the Iceb
 netty = "4.1.112.Final"
 opentelemetry = "1.41.0"
 opentelemetryAlpha = "1.31.0-alpha"
-opentracing = "0.33.0"
```

Comment:
https://github.com/search?q=repo%3Aprojectnessie%2Fnessie%20opentracing&type=code

opentracing is still mentioned in a few places needlessly